fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -61,7 +61,7 @@
     protected static final String SELECT_COUNT_ROOT = "SELECT COUNT(DISTINCT R.RESOURCE_ID) ";
     protected static final String SYSTEM_LEVEL_SELECT_COUNT_ROOT = "SELECT SUM(CNT) ";
     protected static final String SYSTEM_LEVEL_SUBSELECT_COUNT_ROOT = " SELECT COUNT(DISTINCT LR.LOGICAL_RESOURCE_ID) AS CNT ";
-    protected static final String WHERE_CLAUSE_ROOT = "WHERE R.IS_DELETED <> 'Y'";
+    protected static final String WHERE_CLAUSE_ROOT = "WHERE R.IS_DELETED = 'N'";
```

Comment:
so interesting to see how many places this clause is used.

Reply:
Yeah, and ideally this flag should also be at the logical_resource level so that we can know the current status without having the extra join to the _resources table.